modules/serverless-logic-editing-workflows.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-editing-workflows_{context}"]
```

Comment:
![image](https://github.com/openshift/openshift-docs/assets/5937251/e920215f-a39e-4846-8c5b-4c2f480a0361)

I see this file following the `Creating and managing workflow` section and that is plain wrong. The `Editing Workflow` and `Testing workflow` are using `oc` commands yet `kn workflow run` using podman locally and editing and testing workflow in this case is just plain hands-on editing of the workflow and project files.
This has to be adjusted.

Basically it means adding new chapters like:
Creating and managing workflows
Editing workflows locally - does not use `oc`
Testing workflows locally - does not use 'oc' -> just use your favourite editor to edit files

Deploying workflows
Editing deployed workflows - usec `oc`
Testing deployed workflows - uses `oc`

Reply:
good catch, I have missed that part